recipes/fiji-ilastik/post-link.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Version 1.4.0b29
+ilastik_bin=${CONDA_PREFIX}/bin/ilastik
+if [ ! -e "$ilastik_bin" ]; then
+  # Version 1.4.0b27
+  ilastik_bin=${CONDA_PREFIX}/bin/ilastik-app
+  if [ ! -e "$ilastik_bin" ]; then
+    ilastik_bin=""
+  fi
+fi
+
+if [ -n "$ilastik_bin" ]; then
+  echo "
+  ------------------------------------------------------------------------------
+
+  Please run: bash $CONDA_PREFIX/pkgs/fiji-ilastik-1.8.2-0/info/recipe/set_ilastik_executable_in_ImageJ.sh
+    to automatically set the ilastik executable in Fiji in headless mode.
```

Comment:
could this not always be done for the conda local fiji? I think this could fly, with ilastik being a requirement?

Reply:
Not sure I understand what you mean.
My dream would have been that there is something like a post-installation hook that run this bash script after installation.
However, it seems that it is not really possible (https://docs.conda.io/projects/conda-build/en/latest/resources/link-scripts.html).
Maybe this could be part of ilastik4ij in:
https://github.com/ilastik/ilastik4ij/blob/c3f2b21f094f3a66b537f26663fc5d6149a6cfb3/src/main/java/org/ilastik/ilastik4ij/ui/IlastikOptions.java#L23-L25
I don't know how to make it clean.

An alternative way, was to replace `ImageJ` by a script which would first launch the bash script and then run `ImageJ` but this means that the bash script is run at each time which is not ideal (it takes about 20s to run).

Thanks

Reply:
hm I wonder if it is possible somehow to set options via command line? Something like `-Dorg.ilastik.ilastik4ij.executable=...` do you know if that somehow works?

Alternatively one could maybe look into writing the preferences file on post-link...

Reply:
I will check if we can put in the command line. For the preferences file, I checked and it is not written in the preferences file...

Reply:
I don't find how to set the path in the command line... I am really not an expert in Java nor in ImageJ plugin. I think we would need some help here. Maybe @chaubold who initially set this "optionsService": https://github.com/ilastik/ilastik4ij/commit/4b3c5e5a1f8181ddd58b0dd3d45fde1aabb7a461

Reply:
@NicoKiaru, helped me to fix this.